basics-assignment-3-start/basics-assignment-3-start/src/app/app.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { count } from 'rxjs';
+
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+
+
+  // for bootstrap : 
+  constructor(private modalService: NgbModal) {
+  }
+
+  public open(modal: any): void {
```

Comment:
when is this function ever called?

Reply:
it said i needed this function for bootstrap , but i deleted it and the constructor and the bootstrap worked